src/popup/locales/en.json

```diff
@@ -357,20 +360,35 @@
       "channelSlashTx": "Slash channel",
       "channelSettleTx": "Settle channel",
       "channelSnapshotSoloTx": "Solo snapshot channel",
-      "payingForTx": "Paying For Transaction",
+      "payingForTx": "Paying for transaction",
       "pending": "Pending...",
-      "multisigProposal": "Multisig Proposal"
+      "multisigProposal": "Multisig Proposal",
+      "sentTx": "Sending",
+      "receivedTx": "Receiving"
+    },
+    "listType": {
+      "sentTx": "Sent",
+      "receivedTx": "Received",
+      "namePreClaimTx": "Name preclaimed",
+      "nameClaimTx": "Name claimed",
+      "nameUpdateTx": "Name updated",
+      "nameTransferTx": "Name transferred",
+      "nameRevokeTx": "Revoke name",
+      "provide_liquidity": "Liquidity provided",
+      "remove_liquidity": "Liquidity removed",
+      "tipSent": "Tip sent",
+      "tipReceived": "Tip received"
     },
     "dexType": {
-      "provide_liquidity": "provide liquidity",
-      "remove_liquidity":"remove liquidity",
-      "swap": "swap",
-      "allowToken": "allow token",
-      "change_allowance": "allow token",
-      "transfer_allowance": "allow token",
-      "deposit": "deposit",
-      "withdraw": "withdraw",
-      "pool": "pool"
+      "provideLiquidity": "Provide liquidity",
+      "removeLiquidity": "Remove liquidity",
+      "swap": "Swap",
+      "allowToken": "Allow token",
+      "change_allowance": "Allow token",
+      "transfer_allowance": "Allow token",
```

Comment:
Why some of the keys are camelCase and others with snakeCase? Wouldn't that change break transaction confirmation modal?

Reply:
for me all works, we will see after testing

Reply:
Yes, but you should at least stick to one naming convention

Reply:
fixed